portal/views/tou.py

```diff
@@ -11,11 +11,14 @@
 from ..models.audit import Audit
 from ..models.tou import ToU, tou_types
 from ..models.user import current_user, get_user_or_abort
+from .crossdomain import crossdomain
 
 tou_api = Blueprint('tou_api', __name__, url_prefix='/api')
 
 
 @tou_api.route('/tou')
+@crossdomain()
+@oauth.require_oauth()
```

Comment:
Should this require oauth?

Reply:
Nice catch, probably accidentally added; I'll remove